doc_src/build.xml

```diff
@@ -178,12 +218,99 @@
             </classpath>
         </java>
         <copy file="${language}/OmegaT.css" tofile="../docs/${language}/OmegaT.css" />
+    </target>
+    
+    <target name="pictures">
         <copy todir="../docs/${language}/images">
              <fileset dir="${language}/images" />
         </copy>
     </target>
 
-    <target name="html" depends="chunk,instant-start">
+    <target name="html" depends="pictures,chunk,instant-start">
+    </target>
+
+    <target name="whc-toc">
+        <java dir="."
+              fork="true"
+              failonerror="true"
+              maxmemory="128m"
+              classname="com.icl.saxon.StyleSheet" >
+             <arg value="-o" />
+             <arg value="${language}/xhtml5/toc.xml" />
+             <arg value="${language}/xhtml5/index_.html" />
+             <arg value="whc-toc.xsl" />
+             <arg value="language=${language}"/>
+             <classpath>
+               <pathelement location="${saxon}" />
+               <pathelement path="${saxon}" />
+             </classpath>
+        </java>      
+    </target>
+    
+    <target name="whc-index">
+        <java dir="."
+              fork="true"
+              failonerror="true"
+              maxmemory="128m"
+              classname="com.icl.saxon.StyleSheet" >
+             <arg value="-o" />
+             <arg value="${language}/xhtml5/index.html" />
+             <arg value="${language}/xhtml5/index_.html" />
+             <arg value="whc-index.xsl" />
+             <arg value="language=${language}"/>
+             <classpath>
+               <pathelement location="${saxon}" />
+               <pathelement path="${saxon}" />
+             </classpath>
+        </java>
+        <delete file="${language}/xhtml5/index_.html"/>
+    </target>
+    
+    <target name="whc-header">
+        <java dir="."
+              fork="true"
+              failonerror="true"
+              maxmemory="128m"
+              classname="com.icl.saxon.StyleSheet" >
+             <arg value="-o" />
+             <arg value="${language}/xhtml5/header.xhtml" />
+             <arg value="${language}/index.xml" />
+             <arg value="whc-header.xsl" />
+             <arg value="language=${language}"/>
+             <classpath>
+               <pathelement location="${saxon}" />
+               <pathelement path="${saxon}" />
+             </classpath>
+        </java>      
+        <copy file="omegat.svg" tofile="../docs/${language}/images/omegat.svg"/>
```

Comment:
```suggestion
        <copy file="../images/OmegaT.svg" tofile="../docs/${language}/images/omegat.svg"/>
```

Reply:
> We have an existing logo SVG at [images/OmegaT.svg](https://github.com/omegat-org/omegat/blob/master/images/OmegaT.svg); can we not use that rather than add a new logo?

Yes, definitly, I missed it. So you can remove the doc_src/omegat.svg from git

Reply:
Thanks!